pandas/core/indexes/datetimes.py

```diff
@@ -1937,15 +1937,18 @@ def tz_convert(self, tz):
                      mapping={True: 'infer', False: 'raise'})
     def tz_localize(self, tz, ambiguous='raise', errors='raise'):
         """
-        Localize tz-naive DatetimeIndex to given time zone (using
-        pytz/dateutil), or remove timezone from tz-aware DatetimeIndex
+        DatetimeIndex.tz_localize used for Localize tz-naive DatetimeIndex.
+
+	tz_localize method is use to convert naive DatetimeIndex into any other localize
```

Comment:
Your indentation is one space off (this line should be indented the same as the first line of the docstring)